include/benchmark/benchmark.h

```diff
@@ -578,8 +567,8 @@ class State {
     return (max_iterations - total_iterations_ + batch_leftover_);
   }
 
-private: // items we expect on the first cache line (ie 64 bytes of the struct)
-
+ private
```

Comment:
this change looks odd. clang-format misbehaving?

Reply:
Yeah.... I had to move the comment to the next line, otherwise clang-format just kind of mangles the `:` on the `private`.